_data/simple-icons.json

```
@@ -11629,6 +11629,11 @@
             "hex": "6EBE49",
             "source": "https://branding.seagate.com/productpage/3fc51aba-c35a-4eff-a833-a258b0440bd2"
         },
+        {
+            "title": "SearXNG",
+            "hex": "3050ff",
```

```suggestion
            "hex": "3050FF",
```

Our linter complains if the hex code isn't upper-case!